notebooks/vertex_genai/solutions/semantic_search_with_matching_engine_and_palm.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<p>"For the document <code>id</code> we simply use <s>here</s> the row index <em>from</em> the..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/377/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Done!</p>